src/config/hardware.h

```diff
@@ -10,14 +10,16 @@
 #define POT_MAX 1023
 
 // Pins //
-// Interface //
+// Pattern-switching //
```

Comment:
This was the original intent for this section. These will all be deprecated with the new Board abstraction anyway.